### PR TITLE
[WIP] Add discourse information to site

### PR DESCRIPTION
**Pull request by matthiasgeiger**
- [x] add information to help section
- [x] add "Forum" to Header
- [x] add blog post
  - [x] explain reasons for change
  - [x] provide quick start guide (esp. regaring "mailing list mode")
- [x] Add concrete mail adresses
 - [x] to quick start guide
 - [x] to forum categories

After publication:
- [ ] send a mail to the mailing list

~~Should we add a new link to the discourse forum at the header? If yes, how should it be labeled?~~

### CODE REVIEW

**Comment by simonharrer** [comment]
Yes, label ist "Forum".

File: _posts/2016-06-24-JabRefForum.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+However, there we still some things left at sourceforge: 
+Longstanding JabRef user might be aware that there was forum at sourceforge some time ago which has been closed.
+Moreover, there are two mailing lists on for developers, and [one for users](https://lists.sourceforge.net/lists/listinfo/jabref-users).
+
+With the new forum we now again provide an easy to use platform to ask questions (and hopefully to get answers) from the JabRef community.
+Thus, we decided to officially discontinue the support on the mailing list. We'll leave the list open for some time - but the official communcation channels for the future will be the GitHub issue tracker and the discourse forum.
+
+For those of you who like to stick with a mailing-list like behavior we provide you a guide on how to use the discourse forum just like a mailing list.
+
```
Comment (mlep) [inline _posts/2016-06-24-JabRefForum.md]: Could you include the link to this guide?

Comment (matthiasgeiger) [inline _posts/2016-06-24-JabRefForum.md]: Yes, of course. This post is not yet completed... and I also have not written this Guide ;)

**Comment by matthiasgeiger** [comment]
Okay I finished the blog post and the quick start guide, which can be found here: http://discourse.jabref.org/t/quick-start-guide-for-former-mailing-list-users/31

@mlep (and of course all others): It would be great if you could give me some feedback whether this is understandable for someone who has not worked yet with the forum ;-)

**Comment by mlep** [comment]
@matthiasgeiger Both the post and the guide are clear to me. Good job!

**Pull request merged**